sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -1319,13 +1319,19 @@ export interface PathCreateHttpHeaders {
 export interface PathCreateIfNotExistsOptions extends CommonOptions {
     // (undocumented)
     abortSignal?: AbortSignalLike;
+    acl?: PathAccessControlItem[];
     customerProvidedKey?: CpkInfo;
+    expiresOn?: number | Date;
```

Comment:
could we provide convenience of allowing date string? i.e., `expiresOn?: number | string | Date;`

Reply:
To allow string, we'd need to parse it's value (whether a number or a Date).  `Date` has constructor with string parameter, customer can easily get a Date object with a string.  Allowing string type may not provide much convenience to customer.

Reply:
I like your reasoning, @EmmaZhu - let's not introduce ambiguity with how to coerce strings unless developers *really* want this

Reply:
I don't have a strong opinion on this and am fine with current shape. Adding `string` to date input type was Arch Board feedback when we worked on Metrics Advisor.